Makefile

```diff
@@ -9,7 +9,7 @@ env-up-rebuild: env-update-image	## Start devcontainer with rebuild.
 	docker-compose up --build -d
 
 env-update-image:					## Pull latest dev image
-	docker pull "perconalab/pmm-server:dev-latest"
+	docker-compose pull
```

Comment:
It can be like `docker compose pull`, without dash

Reply:
We can, it is compose v2 format, I don't think we need to push for it. It will cause issue for people using linux.